diff --git a/Table/src/lib/AsciiReader.cpp b/Table/src/lib/AsciiReader.cpp
index 53bf750..44a92ad 100644
--- a/Table/src/lib/AsciiReader.cpp
+++ b/Table/src/lib/AsciiReader.cpp
@@ -32,7 +32,12 @@
 using boost::regex;
 using boost::regex_match;
 #include <boost/algorithm/string.hpp>
+
+#if BOOST_VERSION < 107300
 #include <boost/io/detail/quoted_manip.hpp>
+#else
+#include <boost/io/quoted.hpp>
+#endif
 
 #include "ElementsKernel/Exception.h"
 #include "Table/AsciiReader.h"
diff --git a/Table/src/lib/AsciiWriterHelper.cpp b/Table/src/lib/AsciiWriterHelper.cpp
index 1cfc444..3e13f68 100644
--- a/Table/src/lib/AsciiWriterHelper.cpp
+++ b/Table/src/lib/AsciiWriterHelper.cpp
@@ -25,10 +25,15 @@
 #include <algorithm>
 #include <boost/lexical_cast.hpp>
 #include <boost/regex.hpp>
-#include <boost/io/detail/quoted_manip.hpp>
 #include "ElementsKernel/Exception.h"
 #include "AsciiWriterHelper.h"
 
+#if BOOST_VERSION < 107300
+#include <boost/io/detail/quoted_manip.hpp>
+#else
+#include <boost/io/quoted.hpp>
+#endif
+
 namespace Euclid {
 namespace Table {
 
diff --git a/Table/src/lib/AsciiWriterHelper.h b/Table/src/lib/AsciiWriterHelper.h
index 8da8357..6880306 100644
--- a/Table/src/lib/AsciiWriterHelper.h
+++ b/Table/src/lib/AsciiWriterHelper.h
@@ -25,13 +25,17 @@
 #ifndef TABLE_ASCIIWRITERHELPER_H
 #define TABLE_ASCIIWRITERHELPER_H
 
+#include <sstream>
+#include <typeindex>
 #include <vector>
 #include <typeindex>
+#if BOOST_VERSION < 107300
 #include <boost/io/detail/quoted_manip.hpp>
+#else
+#include <boost/io/quoted.hpp>
+#endif
 
 #include "ElementsKernel/Export.h"
-
-
 #include "Table/Table.h"
 
 namespace Euclid {
